addon/helpers/includes.js

```diff
@@ -0,0 +1,7 @@
+import { helper } from '@ember/component/helper';
+
+export function includes([collection, item]/*, hash*/) {
+  return collection.includes(item);
```

Comment:
This needs to guard against collection being null or undefined. There are some things using this template (tutorials, cli guides) that do not use versions at all, and I can see a future where we have versions for things but they won't have LTS versions 🙈